### PR TITLE
fixed cd error return

### PR DESCRIPTION
**Pull request by Leo-Gyger**
cd will now print why it couldn't move to the directory

### CODE REVIEW

**Review by ldominiq — APPROVED** [review]

File: srcs/cd.c [hunk 1]
```diff
@@ -6,12 +6,22 @@
...
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/06/12 17:01:54 by tgrivel           #+#    #+#             */
-/*   Updated: 2022/06/27 17:41:59 by lgyger           ###   ########.fr       */
+/*   Updated: 2022/06/28 12:50:19 by lgyger           ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
+static	void why(char *path)
```
Comment (t-h2o) [inline srcs/cd.c]: Why why ?

File: srcs/cd.c [hunk 2]
```diff
@@ -6,12 +6,27 @@
...
+static void	why(char *path)
+{
+	struct stat	file;
+
+	if (stat(path, &file) == -1)
+	{
+		printf("cd: %s: no such file or directory\n", path);
+		return ;
+	}
+	if (!(file.st_mode & S_IFDIR))
```
Comment (t-h2o) [inline srcs/cd.c:24]: What is that ?
```c
& S_IFDIR
```

File: srcs/cd.c [hunk 3]
```diff
@@ -6,12 +6,27 @@
...
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/06/12 17:01:54 by tgrivel           #+#    #+#             */
-/*   Updated: 2022/06/27 17:41:59 by lgyger           ###   ########.fr       */
+/*   Updated: 2022/06/28 13:10:31 by lgyger           ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
+static void	why(char *path)
```
Comment (t-h2o) [inline srcs/cd.c:15]: Why why?

File: srcs/cd.c (hunk 2 above)
Comment (Leo-Gyger) [reply srcs/cd.c:24]: And will return one if the flag is present in the binary number on the left and S_IFDIR will OR ed in the flag if it is a directionary

**Pull request merged**